ta/arch/arm/link.mk

```diff
@@ -23,6 +23,16 @@ link-ldflags += -z max-page-size=4096 # OP-TEE always uses 4K alignment
 link-ldflags += --as-needed # Do not add dependency on unused shlib
 link-ldflags += $(link-ldflags$(sm))
 
+ifeq ($(CFG_TA_FTRACE_SUPPORT),y)
+dyn_list:
+	$(file > dyn_list,{__ftrace_info;};)
+link-ldflags += --dynamic-list dyn_list
+ftracedep = dyn_list
+cleanfiles += dyn_list
+else
+ftracedep =
+endif
```

Comment:
The `else` part is probably not needed, is it?

Reply:
Not needed, will remove.

Reply:
Done.